CIScripts/Test/Utils/IisTcpTestDockerImageBuild.Tests.ps1

```diff
@@ -0,0 +1,28 @@
+Param (
+    [Parameter(Mandatory=$true)] [string] $TestenvConfFile
+)
+
+. $PSScriptRoot\IisTcpTestDockerImageBuild.ps1
+
+$Sessions = New-RemoteSessions -VMs (Read-TestbedsConfig -Path $TestenvConfFile)
+$Session = $Sessions[0]
+
+Describe "Initialize-IisTcpTestDockerImage" {
+    It "Builds iis-tcptest image" {
```

Comment:
If you decide to make Initialize-IisTcpTestDockerImage more generic, please also test negative case with invalid docker image name.